recipes/dunovo/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+make
+mkdir $PREFIX/include/dunovo
+cp *.py *.so $PREFIX/include/dunovo
```

Comment:
Okay, I changed it to put the `.so` files in `lib` and the executables in `bin`, and edited the Python scripts to find the `.so` files in `lib`.